### PR TITLE
PhpStormStubsSourceStubber: `@since` and `@removed` annotations in some cases do not contain a PHP version, but an extension version - e.g. "@since 1.3.0"

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php [hunk 1]
```diff
@@ -50,6 +50,84 @@ final class PhpStormStubsSourceStubber implements SourceStubber
...
+        'sysvshm',
+        'tidy',
+        'tokenizer',
+        'wddx',
+        'xml',
+        'xmlreader',
+        'xmlrpc',
+        'xmlwriter',
+        'xsl',
+        'Zend OPcache',
```
Comment (Ocramius) [inline src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php:127]: The naming convention across extension is... "sub-optimal" to say it nicely :D

File: test/unit/SourceLocator/SourceStubber/PhpStormStubsSourceStubberTest.php [hunk 2]
```diff
@@ -786,13 +788,16 @@ public function dataConstantInPhpVersion(): array
             ['PHP_OS_FAMILY', 70100, false],
             ['INPUT_SESSION', 70400, true],
             ['INPUT_SESSION', 80000, false],
+            // Not core constants
+            ['RADIUS_DISCONNECT_ACK', null, true],
```
Comment (Ocramius) [inline test/unit/SourceLocator/SourceStubber/PhpStormStubsSourceStubberTest.php:792]: /cc @asgrim look who's back :D

File: src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php [hunk 3]
```diff
@@ -512,12 +601,17 @@ private function isDeprecatedInPhpVersion(Node\Stmt\ClassLike|Node\Stmt\ClassCon
...
     }
 
-    private function isSupportedInPhpVersion(Node\Stmt\ClassLike|Node\Stmt\Function_|Node\Stmt\Const_|Node\Expr\FuncCall|Node\Stmt $node): bool
+    private function isSupportedInPhpVersion(Node\Stmt\ClassLike|Node\Stmt\Function_|Node\Stmt\Const_|Node\Expr\FuncCall|Node\Stmt $node, bool $isCoreExtension): bool
     {
         if ($this->phpVersion === null) {
             return true;
         }
 
+        // We have to check "@since" and "@remove" only for core extensions because there's version of extension for other extensions
```
Comment (Ocramius) [inline src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php]: I really don't understand this comment :-\

**Review by Ocramius — CHANGES_REQUESTED** [review]
Diff seems OK (no evident defects), but I really don't understand the commit message (PR title)

File: src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php (hunk 3 above)
Comment (kukulich) [reply src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php]: https://github.com/JetBrains/phpstorm-stubs/blob/master/radius/radius.php#L50

It's not PHP version, it's version of the extension.

Yeah, my english is bad...

Comment (kukulich) [reply src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php]: Improved.

**Review by Ocramius — APPROVED** [review]
Now I understand this diff, thanks :D

File: src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php (hunk 3 above)
Comment (Ocramius) [reply src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php]: @kukulich at least the code is impeccable ;-)

**Pull request merged**